README.md

```diff
@@ -583,6 +569,135 @@ async function customMiddleware(ctx, next) {
 }
 ```
 
+## Rate Limiting
+
+The Okta API will return 429 responses if too many requests are made within a given time, please see [Rate Limiting at Okta] for a complete list of which endpoints are rate limited.  When a 429 error is received, the `X-Rate-Limit-Reset` header will tell you the time at which you can retry. There are two ways you can handle 429 responses:
```

Comment:
nit: When colons are used, it usually indicates a list follows.

I'd suggest either adding a list to our two headers below, or removing the last sentence of this paragraph.